app/components/agenda/agendaitem/agendaitem-case/agendaitem-titles-edit.hbs

```diff
@@ -45,6 +45,18 @@
       />
     </div>
   </div>
+  <div class="auk-u-mb-4">
+    <div class="auk-form-group">
+      <Auk::Label id="private-comment-title" class="auk-u-icon-left"><Auk::Icon @name="lock-closed" @size="small" /> {{t "private-comment-title"}}</Auk::Label>
```

Comment:
new line for `auk:icon`?
Tried prettier locally, doesn't do a good job for this one.
Manually:
```
<Auk::Label id="private-comment-title" class="auk-u-icon-left">
    <Auk::Icon @name="lock-closed" @size="small" />
    {{t "private-comment-title"}}
</Auk::Label>
```